cloud/pkg/cloudhub/cloudhub.go

```diff
@@ -69,6 +71,9 @@ func (a *cloudHub) Start() {
 	if err := httpserver.PrepareAllCerts(); err != nil {
 		klog.Fatal(err)
 	}
+	// TODO: Will improve in the future
+	DoneTLSTunnelCerts <- true
+	close(DoneTLSTunnelCerts)
```

Comment:
Use unbuffered channel and close it directly? @ls889 @kadisi

Reply:
I use buffer channel here to prevent cloudhub from blocking.

Reply:
If use an unbuffered channel, we must only use the `close（）` method, Instead of doing a channel write 。

I don't think it's a problem for the moment, because the whole piece of logic needs to be modified later。we need independent certs for the tunnel in the future.